_python/site_parser.py

```diff
@@ -0,0 +1,95 @@
+import psycopg2 as ps
+import bs4
+import pandas as pd
+import requests
+
+class Webscrapper:
+
+    def __init__(self, url: str) -> None:
+        self.url = url
+
+    def __start_connections(self) -> ps.extensions.connection:
+        try:
+            connection = ps.connect(
+                host = "db",
+                port = "5432",
+                dbname = "postgres",
+                password = "1234",
+                user = "postgres"
+            )
+
+            return connection
+        except Exception as e:
+            print(f"Error: {e}")
+
+    def __get_data(self) -> list[tuple[str, str, str]]:
+        content = requests.request("get", self.url)
+        soup = bs4.BeautifulSoup(content.text, "html.parser")
+
+        all_lists = soup.find_all('ul', id="", recursive=True)
+        res = []
+
+        for ele in all_lists:
+            if not ele.get('class') and len(ele) == 21:
+               text_list = ele.text.replace("\xa0", "").split("\n")
+
+        for line in text_list:
+            splitted = line.split("— ")
+            splitter = splitted[1].index(",")
+
+            res.append((splitted[0], splitted[1][:splitter], splitted[1][splitter+2:]))
+
+        return res
+    
+    def __construct_df(self) -> pd.DataFrame:
+        site_data = self.__get_data()
+
+        dict_data = {
```

Comment:
nitpick: Could we make it as below?
dict_data = {
    "Date": [f"{date} 2000" for date, _, _ in site_data],
    "Name": [name for _, name, _ in site_data],
    "Info": [info for _, _, info in site_data],
}